### PR TITLE
[WebProfilerBundle] Fix AJAX panel with fetch requests

### PR DESCRIPTION
**Pull request by OnekO**
| Q             | A
| ------------- | ---
| Branch?       | 3.2
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #20723
| License       | MIT
| Doc PR        | None

### CODE REVIEW

**Comment by stof** [comment]
@OnekO I rebased your branch to base it on 3.2 rather than master, as you started the edition in master rather than your target branch.

**Comment by OnekO** [comment]
Thx :)

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig [hunk 1]
```diff
@@ -243,6 +243,17 @@
                 var oldFetch = window.fetch;
                 window.fetch = function () {
                     var promise = oldFetch.apply(this, arguments);
+                    var paramType = Object.prototype.toString.call(arguments[0]);
+                    if (paramType === '[object Request]') {
+                        arguments[0] = arguments[0].url;
```
Comment (OnekO) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig]: You are right, I changed it again.

**Comment by fabpot** [comment]
Thank you @OnekO.

**Pull request closed without merging**